tests.py

```diff
@@ -19,38 +20,74 @@ def test_cage(self):
         cage = Cage()
 
 
-class TestAnimalCreation(unittest.TestCase):
+class TestLionCreation(unittest.TestCase):
```

Comment:
Great tests, but not very DRY.  Can you think of a way of achieving the same tests without repeating yourself?